docs/up-ga/downloads/v1.md

```diff
@@ -11,13 +11,68 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-    No data available for the specified measure.
+  Total downloads of the landing page of a work, as determined by Google Analytics. To count multiple downloads of the same file by the same user, there must be a gap of at least 30 minutes from the last download of that landing page, even if that previous download was not counted.
 </div>
+
+<strong>
+For Example:
+</strong>
+
+<div  style="text-align:center; color:black">
+<table>
+    <tr>
+        <th>Time of download</th>
+        <th>Minutes from last download</th>
+        <th>downloads counted</th>
```

Comment:
For consistency, Downloads should be capitalized

Reply:
`<th>Downloads counted</th>`